target/linux/pistachio/base-files/usr/bin/get_mac.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
Thanks @lperkov for all the comments. Actually we are using https://github.com/CreatorDev/proddata package for saving/retrieving the mac address into OTP of Ci40. All the Ci40-platform related package feeds are hosted in https://github.com/CreatorDev/Ci40-platform-feed. However we are fine to make changes if there is more generic openwrt way of doing such things. Mostly as @hauke has pointed out in other comment, will try to see if we can adopt that style.